docs/content/components/center/space-center.demo.tsx

```diff
@@ -0,0 +1,9 @@
+import { Center } from '@marigold/components';
+
+export default () => (
+  <Center maxWidth="xxlarge" space={1}>
+    <div className="h-28 w-full border border-slate-300 bg-slate-200" />
```

Comment:
they should not be full width, otherwhise you are not seeing them centered :D maybe 50% or somthing?